### PR TITLE
Megachart Unit Test Fix

### PR DESCRIPTION
**Pull request by mlzummo**

### CODE REVIEW

**Comment by mlzummo** [comment]
@GordonSmith @jbrundage  Please Review

**Comment by mlzummo** [comment]
the gmap fix is in another PR

File: src/composite/MegaChart.js [hunk 1]
```diff
@@ -21,13 +20,13 @@
         
         this._layout = new Border();
         
-        this._chart = new MultiChart();
```
Comment (GordonSmith) [inline src/composite/MegaChart.js]: This is correct

File: src/composite/MegaChart.js [hunk 2]
```diff
@@ -21,13 +20,13 @@
...
         this._valueTitle = new Text();
         this._domainTitle = new Text();
     }
     MegaChart.prototype = Object.create(Surface.prototype);
     MegaChart.prototype.constructor = MegaChart;
     MegaChart.prototype._class += " composite_MegaChart";
+    MegaChart.prototype._chart = new MultiChart();
```
Comment (GordonSmith) [inline src/composite/MegaChart.js]: This is not correct

File: src/composite/MegaChart.js [hunk 3]
```diff
@@ -36,7 +35,7 @@
     
     MegaChart.prototype.publish("gutter", 8, "number", "Gap Between Cells",null,{tags:["Basic"]});
     
-    MegaChart.prototype.publish("chartType", "LINE", "set", "Chart Type", _chartTypes.map(function (item) { return item.id; }),{tags:["Basic"]});
+    MegaChart.prototype.publish("chartType", "LINE", "set", "Chart Type", MegaChart.prototype._chart._NDChartTypes.map(function (item) { return item.id; }),{tags:["Basic"]});
```
Comment (GordonSmith) [inline src/composite/MegaChart.js]: Should be:
```MultiChart.prototype._NDChartTypes```

**Comment by mlzummo** [comment]
@GordonSmith Please Re-Review

**Comment by mlzummo** [comment]
still failing one moment...

**Pull request merged**